apps/frontend/src/components/ViewFormsList/ViewFormsList.tsx

```diff
@@ -87,12 +97,23 @@ export default function ViewFormsList() {
   useEffect(() => {
     getForms();
   }, []);
-
+  // event handler for all the checkboozes to a empty list until the boxes are checked then the list appends the checked boxes
+  // keep the hosptial names as long and add the full names to the lists as well
   return (
     <Box p={1}>
       <Center mb={1}>
         <Heading size="xl">Submitted Forms</Heading>
       </Center>
+      <Box>
```

Comment:
Can you add a label above this that says 'filter by state' or something.

Also, maybe put the checkboxes in a dropdown or something so they're not just out on the page. Not necessary though, at some point I think the designers will give us a redesign for this page